packages/@uppy/core/src/UIPlugin.ts

```diff
@@ -195,6 +195,7 @@ class UIPlugin<
 export default UIPlugin
 
 export type PluginTarget<M extends Meta, B extends Body> =
+  | null
```

Comment:
`null` is not a valid `PluginTarget`
```suggestion
```

Reply:
We can probably remove it from the `defaultOptions` if it's `null`, no?

Reply:
I agree we shouldn't accept null. I tried to remove `| null` from PluginTarget and remove `target: null` from `defaultOptions` and it builds fine